src/components/Footer.css

```diff
@@ -20,12 +26,14 @@ footer .icon-container {
   font-size: 1.1em;
 }
 
-footer .icon .footer-icon {
-  max-width: 18px;
-  margin-right: 5px;
+@media all and (min-width: 767px) {
+  footer .icon .footer-icon {
+    max-width: 18px;
+    margin-right: 5px;
+  }
 }
 
-@media all and (max-width: var(--on-palm)) {
+@media all and (max-width: 767px) {
   footer .icon-container {
       display: block;
```

Comment:
since you're already on it, can you add some margin so the icons and descriptions have a little bit more breathing space on mobile?

<img width="262" alt="screen shot 2018-07-10 at 19 01 43" src="https://user-images.githubusercontent.com/330324/42525547-b470aabc-8473-11e8-8990-ba5dc6f30fd0.png">

Reply:
I'll override the margin-bottom for the mobile view